utilities/manual/usermanual_en.html

```diff
@@ -548,14 +548,13 @@ <h1 id="SECTION2">3. Compiling a document</h1>
 <p><IMG src="compile_toolbar.png" border="1" alt="compile_toolbar">
-<p><b>Warning: all your files must have an extension and you can't compile an "untitled" file or a file with a space in its name</b>.</p>
 <h2 id="SECTION23">3.2 The log files</h2>
-<p>With the "Quick Build" command, the log file is automatically displayed in the "Messages / Log file" pannel. While clicking on a number in the "Line" column, the cursor is placed on the corresponding line in the editor and the error is displayed. <br>
-Remark : a summary of the latex errors and warnings is displayed before the full log file.</p>
-<p><IMG src="doc15.png" border="1" alt="doc15"></p>
-<p>The "Next Latex Error"and "Previous LaTeX Error" commands allow to get to the errors detected during compilation.</p>
-<p>Lines with errors, warnings, bad boxes will be highlighted with red, yellow or blue background and you can jump between them using Ctrl+Up/Down. (Ctrl+Shift for errors only, Ctrl+Alt for warnings only, Alt+Shift for bad boxes only)<br>
-A tool tip will show more details of the mistake if you jump to a line (it is also shown if you move the mouse over the mark left from the line numbers).</p>
+<p>The Log tab helps with error handling. It provides the tabs <i>Log File</i> and <i>Issues</i> as well as some buttons to the right (see figure). The Issues tab displays a <i>table of messages</i> taken from the <i>log file</i> displayed below the table.</p>
+<p><IMG src="errorlog.png" border="1" alt="Display of Error Log"></p>
+<p>With the buttons <i>Show Error</i>, <i>Show Warning</i>, <i>Show BadBox</i> (see tooltips) you can choose whether error messages (red), warning messages (yellow) or messages for bad boxes (blue) will be shown. With the button <i>Show Log Markers</i> you can display log marker icons left to the lines in the editor. The tooltips for the log markers show details about the errors if required.</p>
+<p>When there are compile error messages then the log markers will be activated. In addition if option <i>Show log in case of compile error</i> (s. Build settings) is checked then the Log tab will be opened automatically. If option <i>Go to error when displaying log</i> is checked (s. Adv. Editor settings) then the editor's cursor will be placed in the first line which has an error marker.</p>
+<p>You may use the buttons <i>Previous Error</i> and <i>Next Error</i> to toggle back and forth between errors to be jumped. The shortcuts for this are Ctrl+Shift+Up/Down, accordingly Ctrl+Alt+Up/Down and Alt+Shift+Up/Down are used for warnings and bad boxes respectively. You jump unspecifically between markers (including bookmarks) with Ctrl+Up/Down.</p>
```

Comment:
ctrl+up/down is go to prev/next mark. It jumps to the next annotated icon (left of the text) That can be bookmarks but also marked errors or warnings.